src/runner/api/itemRunner.js

```diff
@@ -72,29 +69,28 @@ var itemRunnerFactory = function itemRunnerFactory(providerName, data, options)
     };
 
     //optional params based on type
-    if (_.isPlainObject(providerName)) {
+    if (typeof providerName === 'object') {
         data = providerName;
-        providerName = undefined;
+        providerName = void 0;
```

Comment:
What's your preferred style, `void 0` or `noop`? Because I see both in this PR and it could be nice to one day have consistency on naming.

Reply:
yes we should agree. I was used to use a variable when a lot of usage an `void` when only a few, but we define a convention